PlayerControls/sources/ContentControlsUIProps.swift

```diff
@@ -351,6 +358,8 @@ extension DefaultControlsViewController {
             subtitlesTextLabelHidden = {
                 guard case .player(let player) = props else { return true }
                 guard case .playable(let props) = player.item else { return true }
+                if case .active = props.pictureInPicture { return true }
```

Comment:
I can't design solution with only `guard` for this situation, only with `if` or `switch`
Any ideas?